advanced.ui

```diff
@@ -74,16 +58,42 @@
      </property>
     </widget>
    </item>
-   <item row="2" column="2">
-    <widget class="QLineEdit" name="projSize"/>
+   <item row="5" column="0" colspan="3">
+    <widget class="QCheckBox" name="configCheckBox">
+     <property name="text">
+      <string>Display Camera Configuration on Main Screen</string>
+     </property>
+    </widget>
+   </item>
+   <item row="6" column="0">
+    <widget class="QPushButton" name="showevr">
+     <property name="minimumSize">
+      <size>
+       <width>0</width>
+       <height>33</height>
+      </size>
+     </property>
+     <property name="text">
+      <string>Open Evr</string>
+     </property>
+    </widget>
+   </item>
+   <item row="0" column="2">
+    <widget class="QLineEdit" name="viewWidth"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="calibPVName"/>
    </item>
-   <item row="3" column="0" colspan="2">
-    <widget class="QLabel" name="label_4">
+   <item row="2" column="2">
+    <widget class="QLineEdit" name="projSize"/>
+   </item>
+   <item row="4" column="2">
+    <widget class="QLineEdit" name="displayFormat"/>
+   </item>
+   <item row="4" column="0">
+    <widget class="QLabel" name="label_5">
      <property name="text">
-      <string>Spatial Calibration PV name</string>
+      <string>Display Format</string>
```

Comment:
Maybe "Fit Display Format" or "FWHM Display Format"?  (Yeah, it's used for 1/e^2 width too, but "Display Format" is a little too generic, I think.)

Reply:
I went with the FWHM Display Format, I think its easier to understand.